### PR TITLE
Add a driver for httpbin

### PR DESCRIPTION
**Pull request by felixonmars**
Runs httpbin locally for testing, without the need to access external
httpbin.org instance.

### CODE REVIEW

File: pifpaf/drivers/httpbin.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import sys
+from pifpaf import drivers
+
+
+class HttpBinDriver(drivers.Driver):
+
+    DEFAULT_PORT = 5000
+
+    def __init__(self, port=DEFAULT_PORT, ssl_chain_cert=None, ssl_key=None,
+                 ssl_ca_cert=None, **kwargs):
```
Comment (jd) [inline pifpaf/drivers/httpbin.py:23]: ssl args are not used?

File: setup.cfg [hunk 2]
```diff
@@ -23,6 +23,7 @@ test =
     testtools
     os-testr
     mock
+    httpbin
```
Comment (jd) [inline setup.cfg:26]: This shouldn't be here, it should be installed via tox when testing.

Comment (felixonmars) [reply setup.cfg]: Fixed.

File: pifpaf/drivers/httpbin.py (hunk 1 above)
Comment (felixonmars) [reply pifpaf/drivers/httpbin.py]: Indeed, I overlooked this..

**Comment by jd** [comment]
Still a CI failure though.

**Comment by felixonmars** [comment]
py36 now passes. The py27 failure is irrelevant.

**Pull request merged**